core/rpc/src/author/tests.rs

```diff
@@ -132,6 +135,29 @@ fn should_watch_extrinsic() {
 	);
 }
 
+#[test]
+fn should_return_watch_validation_error() {
```

Comment:
Is that testing the problem that was described?

Reply:
Yes, the test fails on `master` and passes here.